src/components/Select/Select_shim.css

```
@@ -89,6 +121,22 @@ div.neo-multiselect.neo-multiselect--small span.neo-multiselect-combo__header {
   padding-bottom: 0;
 }
 
+.neo-multiselect
+  span.neo-multiselect-combo__header
+  button.neo-multiselect__header.neo-button--width-10px {
+  width: 10px;
+  min-height: 30px;
+  height: 30px; /* same as chip */
+  padding: 0;
+}
+
+.neo-multiselect-combo__header input.neo-input--height-26px {
```

Similar to line 126 above, can be something like:

```suggestion
.neo-multiselect-combo__header span input:first-child {
```